build-system/tasks/helpers.js

```diff
@@ -560,6 +560,7 @@ async function compileJsWithEsbuild(srcDir, srcFilename, destDir, options) {
       outfile: destFile,
       plugins: [plugin],
       minify: options.minify,
+      target: argv.esm ? 'es6' : 'es5',
       incremental: !!options.watch,
```

Comment:
The current code will not be able to rebuild if an error happens during the initial run.

Reply:
Fixed this!